config/application.yml.example

```diff
@@ -10,6 +10,9 @@ APP_HOST: 'lvh.me'
 APP_PROTOCOL: 'http'
 EMAIL_ADDRESS: 'noreply@change.me'
 
+REDIS_HOST: 'localhost'
```

Comment:
Ya que estamos ponelo en los otros, así tenemos un "drop in" si las llegamos a usar. Aunque está el default... para mantener la _simetría_ =P

Reply:
Dale :+1: